bin/git-feature

```diff
@@ -45,19 +60,19 @@ done
 
 concatargs() {
   str=$(IFS='-'; echo "$*")
-  branch="$branch_prefix"/$str
+  branch="$branch_prefix""$branch_separator"$str
 }
 
 if test "${argv[0]}" = "finish"; then
   test -z "${argv[1]}" && echo "$branch_prefix" "<name> required." 1>&2 && exit 1
-  branch="$branch_prefix"/"${argv[1]}"
+  branch="$branch_prefix""$branch_separator""${argv[1]}"
```

Comment:
It would be good to simplify quoting (here, and elsewhere) like so:

```suggestion
  branch="$branch_prefix$branch_separator${argv[1]}"
```

Reply:
Applied.